Frameworks/UIKit/UITableViewCell.mm

```diff
@@ -1517,16 +1523,13 @@ - (void)prepareForReuse {
     }
     _editSelectedViewVisible = FALSE;
 
-    /*
-    if ( _backgroundView != nil ) {
-    [_backgroundView removeFromSuperview];
-    _backgroundView = nil;
-    }
-    if ( _selectedbackgroundView != nil ) {
-    [_selectedbackgroundView removeFromSuperview];
-    _selectedbackgroundView = nil;
+    // for any subivew which isn't internal subview used for building default chrome, need to remove them before re-use
+    for (UIView* subview in [self subviews]) {
```

Comment:
we should confirm whether or not we want to do this for subclasses of UITableViewCell